2.3.1/bintray-apache-couchdb-rpm.repo

```diff
@@ -0,0 +1,6 @@
+[bintray-apache-couchdb-rpm]
```

Comment:
If this is going to land, please move this to the `ubi7/` directory. If it's going to be shared across multiple releases, that could be a new top-level directory if you wanted.

Reply:
Thanks @wohali - this is done with the latest commit. The slightly awkward thing (as per the first commit) is that the docker context needs to be the `2.3.1` directory rather than the directory containing the `Dockerfile`, as Docker doesn't allow importing files from a parent directory.